### PR TITLE
make config ready for zend-expressive and ZF

### PR DESCRIPTION
**Pull request by svycka**
maybe someone has more ideas?

### CODE REVIEW

**Comment by svycka** [comment]
@prolic can you look at coveralls why it no longer posts status hare

**Comment by basz** [comment]
> can you look at coveralls why it no longer posts status hare

Seem it does...? >> "All checks have passed" https://coveralls.io/builds/23447128

File: src/Module.php [hunk 1]
```diff
@@ -21,20 +21,18 @@
 
 namespace ZfcRbac;
 
-/**
- * Class ModuleConfig
- *
- * @author  Florent Blaison <florent.blaison@gmail.com>
```
Comment (basz) [inline src/Module.php]: could you add him to https://github.com/ZF-Commons/zfc-rbac/blob/master/composer.json#L13

File: test/ModuleTest.php [hunk 2]
```diff
@@ -22,28 +22,22 @@
 namespace ZfcRbacTest;
 
 use PHPUnit\Framework\TestCase;
-use ZfcRbac\ModuleConfig;
+use ZfcRbac\ConfigProvider;
+use ZfcRbac\Module;
 
 /**
- * @author  Bas Kamer <baskamer@gmail.com>
```
Comment (basz) [inline test/ModuleTest.php]: him too https://github.com/ZF-Commons/zfc-rbac/blob/master/composer.json#L13 please

File: src/ConfigProvider.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+namespace ZfcRbac;
+
+/**
+ * The configuration provider for the ZfcRbac module
+ *
+ * @see https://docs.zendframework.com/zend-component-installer/
+ */
+final class ConfigProvider
+{
+    /**
```
Comment (basz) [inline src/ConfigProvider.php]: remove doc block

File: src/ConfigProvider.php [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+     */
+    public function __invoke()
+    {
+        return [
+            'dependencies' => $this->getDependencyConfig(),
+            'zfc_rbac' => $this->getModuleConfig(),
+        ];
+    }
+
+    /**
```
Comment (basz) [inline src/ConfigProvider.php]: remove doc block

File: src/ConfigProvider.php [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                \ZfcRbac\Role\InMemoryRoleProvider::class => \ZfcRbac\Container\InMemoryRoleProviderFactory::class,
+                \ZfcRbac\Role\ObjectRepositoryRoleProvider::class => \ZfcRbac\Container\ObjectRepositoryRoleProviderFactory::class,
+                \ZfcRbac\Service\AuthorizationServiceInterface::class => \ZfcRbac\Container\AuthorizationServiceFactory::class,
+                \ZfcRbac\Service\RoleServiceInterface::class => \ZfcRbac\Container\RoleServiceFactory::class,
+                \ZfcRbac\Rbac::class => \Zend\ServiceManager\Factory\InvokableFactory::class,
+            ],
+        ];
+    }
+
+    /**
```
Comment (basz) [inline src/ConfigProvider.php]: remove doc block

**Comment by svycka** [comment]
ping @prolic

**Pull request merged**